app/Http/Controllers/ImagesController.php

```diff
@@ -102,6 +104,17 @@ public function update($id, Request $request)
         $originalImage = $this->aws->storeImageData($originalImage->__toString(), $originalFilename);
         $editedImage = $this->aws->storeImageData((string) $editedImage, 'edited_' . $post->id);
 
+        if (config('features.glide') && ! empty(config('services.fastly'))) {
```

Comment:
@DFurnes I added the extra check - wanted to make sure this is what you meant before merging.

Reply:
Will this array itself ever be empty (since it has the nulled keys `['url' => null, 'key' => null]`)? It might make more sense to specifically check that none of those keys are empty.

I imagine we'd want also this in other places where we interact with Fastly too, so is this maybe something that makes sense to move into the `purgeKey` method itself? (So we always call `purgeKey` but short-circuit it if the client isn't fully configured?)

Reply:
ahh yea. That is a great idea!